ci-docker-base/Dockerfile.cilint-clang-tidy

```
@@ -10,7 +10,7 @@ RUN apt-add-repository ppa:git-core/ppa
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     git python3-dev python3-pip python3-setuptools python3-wheel build-essential time wget \
-    cmake clang lld ninja-build
+    cmake clang lld ninja-build libomp-dev
```

Since the rest of our llvm tooling is v11, to be consistent, we can install v11 of the Open MP runtime.
```suggestion
    cmake clang lld ninja-build libomp-11-dev
```